app-backend/db/src/test/scala/com/azavea/rf/database/UserGroupRoleDaoSpec.scala

```diff
@@ -57,6 +58,35 @@ class UserGroupRoleDaoSpec extends FunSuite with Matchers with Checkers with DBT
     }
   }
 
+  test("list users by group") {
+    check {
+      forAll {
+        (userCreate: User.Create, manager: User.Create, orgCreate: Organization.Create, teamCreate: Team.Create,
+         platformCreate: Platform.Create, ugrCreate: UserGroupRole.Create, ugrUpdate: UserGroupRole.Create, page: PageRequest) => {
```

Comment:
I think the compiler might not be happy when running tests since `Platform.Create` was removed in the anchor branch. We could just use `Platform` here.